sufia-models/app/jobs/copy_permissions_job.rb

```diff
@@ -0,0 +1,23 @@
+class CopyPermissionsJob 
+  def queue_name
+    :permissions
+  end
+
+  attr_accessor :id
+
+  def initialize(id)
+    self.id = id
+  end
+
+  def run
+    work = ActiveFedora::Base.load_instance_from_solr(id)
+    if work.respond_to?(:generic_files)
+      work.generic_files.each do |file|
+        file.visibility = work.visibility
+        file.edit_users = work.edit_users
```

Comment:
@jcoyne Is there a simple way to copy permissions from one Hydra object to another? @scherztc I believe there are additional permissions missing in the copy here.  view_users & view groups I know are missing.  I'm hoping there is a AF or Hydra method to get everything.